### PR TITLE
[Fix] Fix dataloader for ncnn quantization

### PR DESCRIPTION
**Pull request by irexyc**
## Motivation

fix https://github.com/open-mmlab/mmdeploy/issues/1991

## Modification

dataloader for ppq

### CODE REVIEW

**Review by RunningLeon — APPROVED** [review]
LGTM

**Comment by RunningLeon** [comment]
@tpoisonooo @AllentDan any comment on this PR?

File: tools/onnx2ncnn_quant_table.py [hunk 1]
```diff
@@ -44,16 +47,10 @@ def get_table(onnx_path: str,
...
-        if isinstance(input_tensor, list):
-            input_shape = input_tensor[0].shape
-            collate_fn = lambda x: data_preprocessor(x[0])[0].to(  # noqa: E731
-                device)
-        else:
-            input_shape = input_tensor.shape
-            collate_fn = lambda x: data_preprocessor(x)[0].to(  # noqa: E731
-                device)
-        break
+        input_tensor = input_data['inputs']
```
Comment (tpoisonooo) [inline tools/onnx2ncnn_quant_table.py:50]: In some old testcase, `input_tensor` is `list`.

**Review by tpoisonooo — APPROVED** [review]

File: tools/onnx2ncnn_quant_table.py (hunk 1 above)
Comment (RunningLeon) [reply tools/onnx2ncnn_quant_table.py:50]: @irexyc Did we only test on mmcls models? Could we test on models from other codebases as well?

**Review by AllentDan — APPROVED** [review]
LGTM after resolving comments.

File: tools/onnx2ncnn_quant_table.py (hunk 1 above)
Comment (RunningLeon) [reply tools/onnx2ncnn_quant_table.py:50]: tested ok on mmpose.

**Pull request merged**